src/main/resources/webroot/monitoring.html

```diff
@@ -213,6 +213,6 @@
     {{/each}}
 </script>
 
-<script src="custom-js/monitoring.js"></script>
+<script src="custom-js/monitoring.js?v=1.0.2"></script>
```

Comment:
Ah, gotcha - my Javascript is very rusty.  Maybe change this to `v=1.0.1` instead then?